docs/faq.md

```diff
@@ -161,13 +160,43 @@ See [the Semgrep development philosophy](/contributing/semgrep-philosophy/) for
 
 ### Comparing Semgrep to Snyk
 
-For SCA, reachability analysis is critical to cut down noise and reduce false positives. Semgrep offers reachability analysis for [several languages, such as Java, JavaScript, and Ruby](supported-languages/#semgrep-supply-chain) whereas Snyk only offers reachability for Java.
+#### SAST
+
+Both Semgrep and Snyk offer out-of-the-box SAST solutions. Semgrep makes it easier to customize the rules that run against your code. Because these rules are visible and customizable, you can analyze your results to see if the relevant vulnerabilities were caught.
+
+In addition to selecting your rules, Semgrep allows you to write custom rules to capture use cases driven by your organization's goals. To help you write rules, [Semgrep Editor](https://semgrep.dev/playground) provides a structure mode to guide you through the process, allows you to test your in-progress rules, and adds them to your organization’s [Policies page](/semgrep-code/policies). Semgrep offers rule-writing capabilities to all users, while Snyk limits it to Enterprise users.
+
+Both Semgrep and Snyk offer remediation advice for findings identified during scans. Snyk displays its recommendations in its UI and CLI, while Semgrep displays remediation advice and guidance in its UI, CLI, and in the form of PR or MR comments.
```

Comment:
```suggestion
Both Semgrep and Snyk offer remediation advice for findings identified during scans. Snyk displays its recommendations in its web app, in supported IDEs, and CLI, while Semgrep displays remediation advice and guidance in its web app, supported IDEs, the CLI, and in the form of PR or MR comments.
```
Just today we found out that they're [going to show AI recommendations in the IDE](https://semgrepinc.slack.com/archives/C05GY0PH79R/p1726110733326119). Because this statement is general, we can also say that we have IDE recommendations (we do, human-written autofix is available in our VSCode plugin).

Reply:
It does seem like a pretty clunky sentence now that we have so many list items though :sweat_smile:  sorry about that.